migreringer/src/main/resources/db/postgres/defaultDS/1.0/V1.0_049__behandling_vedtaksbrev.sql

```diff
@@ -0,0 +1,29 @@
+alter table brevbestilling
+    alter column mottaker_id drop not null;
+
+alter table brevbestilling
+    alter column mottaker_id_type drop not null;
+
+create sequence if not exists seq_behandling_vedtaksbrev start with 1 increment by 50;
```

Comment:
Dei fleste andre sekvenser for id begynner på 1000000. Tore har forklart meg kvifor ein gang...

Reply:
hehe ok. Fiksa også for en annen tabell jeg laget en gang ....